addons/l10n_hu_edi/models/account_move.py

```diff
@@ -390,6 +393,16 @@ def _l10n_hu_edi_check_invoices(self):
                 'message': _('The following invoices appear to be earlier in the chain, but have not yet been sent. Please send them first.'),
                 'action_text': _('View invoice(s)'),
             },
+            'invoice_advance_not_paid': {
+                'records': advance_invoices.filtered(
+                    lambda m: (
+                        m.payment_state not in ['in_payment', 'paid', 'partial']
+                        or m.l10n_hu_edi_state in [False, 'rejected', 'cancelled'] and m not in self
```

Comment:
(as discussed, it's fine to send both an advance and a final invoice at the same time, since we sort them by ID before sending.) I've added a comment to this effect. :smile: